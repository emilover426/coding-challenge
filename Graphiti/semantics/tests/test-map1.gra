int main() {
    map m;
    m = {["Michal":"Language Guru","Alice":"System Architect"]};
    printm(m);
    return 0;
}
