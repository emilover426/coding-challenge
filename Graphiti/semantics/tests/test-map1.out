{ 
	"Michal" : "Language Guru",
	"Alice" : "System Architect" 
}
